Start-C4bNexusSetup.ps1

```diff
@@ -91,21 +92,28 @@ process {
     # Add ChocolateyInternal as a source repository
     choco source add -n 'ChocolateyInternal' -s "$((Get-NexusRepository -Name 'ChocolateyInternal').url)/index.json" --priority 1
 
+    # Add ChocolateyCore as a source repository
+    choco source add -n 'ChocolateyCore' -s "$((Get-NexusRepository -Name 'ChocolateyCore').url)/index.json" --priority 0 --admin-only
+
     # Install a non-IE browser for browsing the Nexus web portal.
-    if (-not (Test-Path 'C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe')) {
+    if (Test-Path 'C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe'){
+        Write-Host "Syncing Microsoft Edge, to bring it under Chocolatey management"
+        choco sync --id="Microsoft Edge" --package-id="microsoft-edge"
```

Comment:
Server 2019 does not ship with Microsoft Edge

Reply:
Yes, we should. I'm just warning that in cases where Edge has been installed via servicing channel (think a Windows Update), choco sync does _not_ detect it, even though it's installed, so we might drop out of the check erroneously.

It might be simpler to just go "screw it, I'm putting edge on here", install the package, and let the underlying install sort things out.

I bring this up because I've _seen_ sync not work even though Edge was _very clearly_ displaying in Programs and Features. It's a real peach.